src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -368,6 +368,15 @@ public bool IsVisible
             }
         }
 
+        /// <summary>
+        /// Determines if node view is visible in canvas or not.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsNodeViewHidden
+        {
+            get => NodeModel.IsNodeViewHidden;
```

Comment:
I think not all properties from ViewModel needs to be popped down to NodeModel, maybe we can leave it on this layer? NodeViewModel is already the datacontext for NodeView, so that should be doable?

Reply:
Yes, makes sense, I'll make adjustments.